Year summary

pjazdzewski1990 opened the pull request.

pjazdzewski1990 commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,42 @@
+- opened an office in San Francisco
+- helped **9** clients in building awesome products
+- open sourced **3** of our projects
+- **X** candidates applied for a position in Scalac
```
I'm still waiting on this number - it will be filled in or deleted

kubukoz replied:
I would remove the "we" in line 19 - the first point is not a "we" and in the 4th there is another "we".

pjazdzewski1990 replied:
Frankly, I would rather rephrase these two bullet points ...

tomasz-biernacki commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,42 @@
+layout: post
+slug: 2015-summary
+title: How Scalac changed in 2015
+summary: End of the year is a great time to sum up the thing that changed in the last months. This post explains how Scalac changed in 2015.
```
The end of the year?

tomasz-biernacki replied:
the things

tomasz-biernacki commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,42 @@
+- Scalac
+---
+
+Scalac is a new company, founded just recently in early 2014. We didn't even have our second birthday ... yet. This post is about us celebrating our small victories. We are looking back on 2015 to recognise things we did good, things that could be improved and learn from things that didn't go as planned at all. 
```
"we did *well*" sounds better to me but feel free to double check

kubukoz replied:
"the things we did good"?

tomasz-biernacki replied:
Not 100% sure but this sound better to me (feel free to disagree):
"We haven’t even had our 2nd birthday yet."

kubukoz replied:
^ :+1:

marioosh replied:
recognise -> recognize

pjazdzewski1990 replied:
I will rewrite it to "We are looking back on 2015 to recognize the things we did well ..."
When it comes to recognise vs recognize, both forms are correct and mean the same things. IMO the issue is just American vs. British English. I will go with `z` to keep it coherent

tomasz-biernacki commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,42 @@
+
+## <a name="summary" href="#summary">Summary</a>
+
+As you can see 2015 was a busy year for us. We hope the next year will be as good. 
```
"...as good as this one" or "...next year will be good as well"

pjazdzewski1990 replied:
The first one is closer to the original meaning

kubukoz commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,42 @@
+- **X** candidates applied for a position in Scalac
+- **1** person took part in our apprenticeship program we started just recently in November
+- **2** people took part in [CODE 2.0](http://code.pfig.org.pl/) internship program at Scalac
+- our Twitter followers grew by **384**
```
Twitter followers _count_?

marioosh commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,42 @@
+title: How Scalac changed in 2015
+summary: End of the year is a great time to sum up the thing that changed in the last months. This post explains how Scalac changed in 2015.
+author: Patryk Jażdżewski
+description: End of the year is a great time to sum up the thing that changed in the last months. This post explains how Scalac changed in 2015.
```
thing -> things

pjazdzewski1990 commented:
@marioosh @kubukoz @tomasz-biernacki I've addressed the remarks. If nothing comes up I will merge it around noon, but feel free to comment if there's something still bothering you. I will fix it right away :) Thanks

tomasz-biernacki commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,41 @@
+layout: post
+slug: 2015-summary
+title: How Scalac changed in 2015
+summary: The end of a year is a great time to sum up changes. This post explains how Scalac changed in 2015.
```
"The end of *the* year" was better for me

tomasz-biernacki commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,41 @@
+title: How Scalac changed in 2015
+summary: The end of a year is a great time to sum up changes. This post explains how Scalac changed in 2015.
+author: Patryk Jażdżewski
+description: The end of a year is a great time to sum up changes. This post explains how Scalac changed in 2015.
```
Same as above

tomasz-biernacki commented:
@pjazdzewski1990 For me just one change and it's :+1:

leszekgruchala commented on _posts/2015-12-31-year_summary.md:
```diff
@@ -0,0 +1,41 @@
+- Scalac
+---
+
+Scalac is a new company, founded just recently in early 2014. We haven’t even had our 2nd birthday ... yet. This post is about us celebrating our small victories. We are looking back on 2015 to recognize the things we did well, things that could be improved and learn from things that didn't go as planned at all. 
```
`birthday ...` -> `birthday...`

leszekgruchala replied:
`We are looking back on 2015 to recognize the things we did well, things that could be improved and learn from things that didn't go as planned at all. `

pjazdzewski1990 replied:
The statement above is true - we are doing that, but we aren't sharing all of them here ;) 
I will change the sentence order here - so "celebrating small victories" will be last.
